Modules/binascii.c

```diff
@@ -76,7 +76,8 @@ get_binascii_state(PyObject *module)
 }
 
 
-static const unsigned char table_a2b_base64[] = {
+/* Align to 64 bytes to ensure table fits in a single L1 cache line */
```

Comment:
The entire 256-bytes table will not fit in a single L1 cache line.

It may be worth to align anyway, but the comment is incorrect.